governance/chat/program/tests/program_test/mod.rs

```diff
@@ -33,11 +35,33 @@ pub struct GovernanceChatProgramTest {
     pub bench: ProgramTestBench,
     pub program_id: Pubkey,
     pub governance_program_id: Pubkey,
+    pub voter_weight_addin_id: Option<Pubkey>,
 }
 
 impl GovernanceChatProgramTest {
+    #[allow(dead_code)]
     pub async fn start_new() -> Self {
+        Self::start_impl(false).await
+    }
+
+    #[allow(dead_code)]
+    pub async fn start_with_voter_weight_addin() -> Self {
+        ensure_voter_weight_addin_is_built();
+
+        Self::start_impl(true).await
+    }
+
+    #[allow(dead_code)]
+    async fn start_impl(use_voter_weight_addin: bool) -> Self {
         let mut program_test = ProgramTest::default();
+
+        solana_logger::setup_with_default(
+            "solana_rbpf::vm=debug,\
+             solana_runtime::message_processor=debug,\
+             solana_program_runtime=debug,\
+             solana_program_test=info",
+        );
```

Comment:
Is this still needed? It was supposed to be fixed in https://github.com/solana-labs/solana/pull/22133/files